doc/datatable.md

```diff
@@ -1,27 +1,36 @@
 
+Some props can be obtained from configuration of the project  (e.g. in demo app `headers` and `limit` props are obtained from the configuration).
+
 Prop `title` is an object that has obligatory property `text` which is string and it will be rendered as table title. Also `title` has optional property `icon` for rendering icon alongside the table title and which can be either React Component (e.g. from package `@material-ui/icons`) or it can also be a string which is classname for icon element and will be rendered as <i className={icon}></i>
 
-Prop `headers` is basically an array containing objects with obligatory properties `text` and `type`. Property `text` is a string that will be rendered as a header (first letter of `headers.text` automaticly will be converted to uppercase) in headers row of the table. Property `type` is a type of data in each cell for column of that header, it can be either `string` or `json`.
-Order of headers in a table is the same as in prop `headers`.
+Prop `headers` is basically an array containing objects with obligatory properties `name` and `key`. Property `name` is a string that will be rendered as a header cell in headers row of the table. Property `key` is a key name for getting data from `data` prop, it must be the same as it is in objects in `data` prop.
+Order of headers in a table is the same as it is in prop `headers`.
 
-Prop `data` is an array of objects. `DataTable` goes through `data` and looks for those properties which has the same name as 
-`headers[idx].text`, if such exist, it will be rendered under that `headers[idx].text` as string or json.
+Prop `data` is an array of objects. `DataTable` component goes through objects in `data` and for each object looks for those properties which has the same key name that was defined in `headers[idx]`. If properties exist in that object from `data`, they will be rendered as string under the appropriate headers. Otherwise if some properties doesn't exist in that object, the `-` symbol will be rendered instead.
+Prop's `data` length should be less or equal to 10 or to `limit` prop if such is defined. If `data` length is more than `limit`, then it will be cutted. 
+Array `data` may be fetched from `axios` in container of the `DataTable` for example. In demo we used predefined `initData` object contains data collection and `fetch(limit, page, str)` method that accepted Table's limit, current page and string from search input field. The `fetch` method is immitating fetch request from the server. You may take a look at our demo app to see how data for `data` prop is initialized and updated.
 
 Also `DataTable` needs props `count`, `currentPage` and `setPage`. Prop `count` is number representing the count of all items, it is used to calculate last page in pagination of `DataTable`. Props `currentPage` and `setPage` are number and function respectively, first represents current page of `DataTable`, second is used to navigate between pages in `DataTable`.
```

Comment:
@titkov49 I would also write here, that the amount of data for pagination can be obtained from API endpoint for every page when set as a parameter in the request, e.g.

```
Axios.get("", {params: {p:page, i: limit, f: search}})
```

where `f` parameter is the search or filter value for `search` prop